bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md

```diff
@@ -64,3 +64,16 @@
 2. Check if auth credentials are correct.
 3. Check if server is running.
 
+## Limitations and issues
+
+There are known limitations and issues that are tracked by OpenSearch including the items listed below.
+
+| Issue | Description |
+| ---- | ----------- |
+| Visualizations without numerical columns | Visualizations will only work if there is a numerical column included due to a known limitation in OpenSearch regarding subqueries. An exception to this would be visualizations with columns only of type string works as well. Can be tracked with this [bug](https://github.com/opensearch-project/sql/issues/347) |
+| Visualizations with First or Last options selected do not work | <img src="img/pbi_issue_first_last.png" width="200"> |

```

Comment:
These don't render correctly. But it may just be a github pr issue. Thought I would suggest we avoid pngs here as many developers would likely prefer text only.

Reply:
Updated in e89be25a0afeb0eb8dbe366e461a19f7144c9b6b